core/src/main/java/edu/wpi/grip/core/events/ExceptionEvent.java

```diff
@@ -22,7 +23,7 @@
      * @param message   The message associated with this event.
      *                  If <tt>null</tt> will use {@link Exception#getMessage()}
      */
-    public ExceptionEvent(Object origin, Exception exception, String message) {
+    public ExceptionEvent(Object origin, @Nullable Exception exception, @Nullable String message) {
```

Comment:
What?  Since when did we use a framework that provides `@Nullable`?  And why are you even modifying this class in this PR?